src/OpenDiffix.Core.Tests/Aggregator.Tests.fs

```diff
@@ -39,7 +43,7 @@ let makeAnonArgs hasValueArg random numAids length =
 let makeStandardArgs hasValueArg random length =
   (if hasValueArg then
      // Generates a sequence of [ Value.Integer int ]
-     buildIntegerSequence random |> Seq.map (fun argValue -> [ argValue ])
+     buildRealSequence random |> Seq.map (fun argValue -> [ argValue ])
```

Comment:
I wouldn't be surprised if the associative laws of merging don't apply if real numbers are at play because of precision losses.

Reply:
Hm, is there anything we should do about this?

Reply:
If the tests pass then no. In the future if it happens then we can add a small tolerance.